Makefile

```diff
@@ -125,7 +125,7 @@ biolink-model.proto: biolink-model.yaml env.lock
 # RDF
 # ~~~~~~~~~~~~~~~~~~~~
 biolink-model.ttl: biolink-model.yaml env.lock
-	pipenv run gen-rdf -f ttl --context https://w3id.org/biolink/biolink-model/context.jsonld $<  > $@
+	pipenv run gen-rdf -f ttl --context https://w3id.org/biolink/biolinkml/context.jsonld $<  > $@
```

Comment:
I don't understand this

Reply:
The biolinkml context.jsonld is used to transform an instance of a biolinkml model into RDF, biolink-model being an example.  The biolink-model context.jsonld is used to transform an instance of _biolink-model_ into RDF .  We erroneously changed the above line to correct an issue with `subproperty_of`, where the RDF generation was failing because the subproperty_of's in the biolink model weren't URI's.   The switch stopped converting them, but it also stopped converting about 90% of the rest of the model.